airbyte-ci/connectors/metadata_service/lib/metadata_service/commands.py

```diff
@@ -61,3 +61,14 @@ def upload(metadata_file_path: pathlib.Path, doc_path: pathlib.Path, bucket_name
     else:
         click.secho(f"The metadata file {metadata_file_path} was not uploaded.", color="yellow")
         exit(5)
+
+@metadata_service.command(help="Upload docs for all connectors to a GCS bucket.")
+@click.argument("connectors-dir", type=click.Path(exists=True, path_type=pathlib.Path))
+@click.argument("docs-dir", type=click.Path(exists=True, path_type=pathlib.Path))
+@click.argument("bucket-name", type=click.STRING)
+def upload_all_docs(connectors_dir: pathlib.Path, docs_dir: pathlib.Path, bucket_name: str):
```

Comment:
❗Lets be clear this is a dev command by prefixing the command with experimental: `experimental_upload_all_docs`